config/rbac/role.yaml

```diff
@@ -56,3 +56,11 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Should we add [kubebuilder](https://github.com/redhat-appstudio/application-service/blob/75a4c79e49ab5c1a4c15d844256d1e4419da63e3/controllers/component_controller.go#L67) annotations to automatically generate this configuration?

Reply:
I think it would be better than adding permissions manually.

Reply:
Let's fix this in a separate PR, then